eng/pipelines/runtime.yml

```diff
@@ -0,0 +1,138 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+    - master
+    - release/*.*
+
+jobs:
+#
+# Checkout repository
+#
+- template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build CoreCLR
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: checked
+    platforms:
+    - OSX_x64
+    - Linux_x64
+    - Linux_arm
+    - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm
+    - Windows_NT_arm64
+    jobParameters:
+      testGroup: innerloop
+
+#
+# Build libraries using live CoreLib from CoreCLR
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - OSX_x64
+    - Linux_x64
+    - Linux_arm
+    - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm
+    - Windows_NT_arm64
+    jobParameters:
+      isOfficialBuild: false
+      testScope: empty
```

Comment:
No need to pass this down here. The default is already `''` . This is just used when running the tests as part of the build-job.